content/en/docs/setup/install/helm/index.md

```diff
@@ -19,6 +19,14 @@ window.onload = function(){
 }
 </script>
 
+<script id="dnscerts" defer>
```

Comment:
What is this JavaScript here for? This doesn't seem right at all.

Reply:
This seems to be the same trick I did for CNI. When linking from another page, if the URL ends with e.g. `#cni`, it switches to a particular tab on the page. So it's less confusing for viewers.

This Javascript is fine, but it must be merged into the JS function above or it will break the functionality for CNI as it is.

Reply:
Done: it has been merged into the JS function above.

Reply:
@geeknoid @rlenglet please take another look.

Reply:
The empty div tag is added to fix the lint error.